frontend/src/views/labels/ListLabels.vue

```diff
@@ -33,21 +33,19 @@
 					v-for="label in labelStore.labelsArray"
 					:key="label.id"
 					:class="{'disabled': userInfo.id !== label.createdBy.id}"
-					:style="getLabelStyles(label)"
-					class="tag"
+					class="label-item"
 				>
-					<span
-						v-if="userInfo.id !== label.createdBy.id"
-						v-tooltip.bottom="$t('label.edit.forbidden')"
-					>
-						{{ label.title }}
-					</span>
+					<XLabel
+						:label="label"
+						:clickable="true"
+					/>
 					<BaseButton
-						v-else
+						v-if="userInfo.id === label.createdBy.id"
```

Comment:
move the edit button to the place of the delete button and remove the delete button

adjust the styling as well